package.json

```diff
@@ -26,7 +26,7 @@
     ]
   },
   "scripts": {
-    "zksync": "yarn workspace zksync",
+    "zksync": "yarn workspace rif-aggregation",
```

Comment:
have we checked that by changing the workspace name it doesn't break the way we run the commands? I ask you this because I'm not familiar with yarn workspaces

Reply:
Actually its needed to be found. Makes reference to the [name](https://github.com/rsksmart/ri-aggregation/blob/rsk_merge_master_Dec2021/sdk/zksync.js/package.json#L2) defined on the package.json of the zksync-js package

Reply:
Updated